proto/kvrpcpb.proto

```diff
@@ -858,3 +858,9 @@ message KeyRange {
     bytes start_key = 1;
     bytes end_key = 2;
 }
+
+enum ExtraRead {
+    Noop = 0;
+    Deleted = 1;
+    Updated = 2;
```

Comment:
Does `Deleted` mean only `delete` event contains the previous value, and `Updated` mean `put` event (insert or update) contains the previous value?
Maybe we need a mode that both `delete` and `put` contain the previous value.